srcpkgs/cnping/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'cnping'
+pkgname=cnping
+version=1.0b2
+revision=1
+hostmakedepends="gcc make libcap-progs"
+makedepends="libXinerama-devel libX11-devel"
+short_desc="Minimal Graphical IPV4 Ping Tool"
+maintainer="Jan Christian Grünhage <jan.christian@gruenhage.xyz>"
+license="MIT"
+homepage="https://github.com/cnlohr/cnping"
+distfiles="https://github.com/cnlohr/cnping/archive/${version}.tar.gz"
+checksum=6dcdfe42b860ad57a2b10dc79a23e5d6af721a3677888af983fd0dea34a0f811
+
+do_build() {
+	make
+	setcap cap_net_raw=ep cnping
```

Comment:
I think setcap needs to be moved to an `INSTALL` script, see `srcpkgs/extrace/INSTALL` as example.